api/api.proto

```diff
@@ -4204,7 +4204,7 @@ message SdkVersion {
     // SDK version major value of this specification
     Major = 0;
     // SDK version minor value of this specification
-    Minor = 77;
+    Minor = 79;
```

Comment:
In SDK_CHANGELOG.md there already exists a note on release 0.78. I am assuming someone forgot to update revision, so skipping 78, and bumped revision to 79.